modules/oe_theme_content_entity_contact/oe_theme_content_entity_contact.module

```diff
@@ -23,7 +23,7 @@ function oe_theme_content_entity_contact_oe_contact_view_alter(array &$build, Co
     ];
 
     foreach ($label_overrides as $name => $label_override) {
-      if (!$entity->get($name)->isEmpty() && isset($build[$name]['#title'])) {
+      if ($entity->hasField($name) && !$entity->get($name)->isEmpty() && isset($build[$name]['#title'])) {
```

Comment:
Because here we are assuming the fields are there to change, but some contact bundles might not have those fields (like the organisation bundle we introduce are themign here).  For the other entity alters we check for the bundle, which is an alternative option if you rather do it that way.

Reply:
Right sorry, I didn't pay enough attention to it.